recipes/libcugraph/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Derived from cugraph build script, as seen here:
+# https://github.com/rapidsai/cugraph/blob/db20b485cfc5399214afcff604b38493f38e83bf/build.sh#L137
+
+CUGRAPH_SRC_DIR=FIXME: set this!
```

Comment:
Using the folders specified in `meta.yaml`, we can do this

```suggestion
CUGRAPH_SRC_DIR="${SRC_DIR}/cugraph"
RMM_ROOT="${SRC_DIR}/rmm"
```

Reply:
Fixed in next commit.